packages/deploy/release-script/release-script.ts

```diff
@@ -249,6 +253,9 @@ const checkIfTooLate = async () => {
     }
     return result
   }
+const runTests= async () => {
```

Comment:
formatting off?

interesting, how this happened, can you check if linter works?